priv/data.yml

```diff
@@ -1,2 +1,2 @@
-- Lorem ipsum
-- Dolor sit amet
+- "ae meu princesos e principas, tudo bem? Vocês sabiam que em 18 de dezembro de 1954 foi fundado no interio do Rio Grande do Sul o município de Não-Me-Toque? Este que tem apenas 17mil habitantes e um pib de aprox. R$ 462mil, aqui na :ptec:  nós ja fechamos contratos maiores, faturamos mais que uma cidade toda! Não-Me-Toque também me lembra do classico noventista You Cant Touch This do  MC Hammer que claramente foi inspirado pela pequena cidade rio grandense. Sabe oque é 18 também mas não é para dezembro? O review do meu PR, 18 linhas só po :gato_de_botas:"
+- "Meus nobres, voces sabiam que a 4000 anos atras, a humanidade na Mesopotamia (do Grego, terra entre Rios) desenvolvia com simbolos cuneiformes a matemática de base 60 ela foi usada principalmente para representar tempo e expecula-se que um dos principais motivos era a possibilidade de contar até 12 com apenas uma mão (pode tentar agora!! Use seu dedão para contar cada gomo dos seus outros 4 dedos). Sabe o que mais é 4000?? O ID do meu novo PR no webapp que está aguardando o seu review ♥"
```

Comment:
This is another case of a very specific case, you can see it here:
```Sabe o que mais é 4000? O ID do meu novo PR no webapp que está aguardando o seu review ♥```

It just makes sense if your PR have an ID higher than 4000. But we can think in another thing to make it less specific. Any ideas @Cendrao ?
And we need to remove this `webapp` as well 😆

Reply:
Is there a way to make a little customisable? 

Like a file with cool facts list that is grouped by years, them we can select the facts based in the PR number comparing with the that year, or others things like:
- the difference of current year to fact year is the same number of PR
- the fact year is the same of the PR number
- the PR number is multiple of fact year

It's kinda crazy suggestion I know, but it would be very nice 😆